cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -11,7 +11,17 @@
 
 
 def approx_eq_mod(a: cirq.TParamVal, b: cirq.TParamVal, period: float, atol: float = 1e-8) -> bool:
-    """Check if a ~= b (mod period). If either input is an unresolved parameter, returns a == b."""
+    """Check if a ~= b (mod period). If either input is an unresolved parameter, returns a == b.
```

Comment:
Given that this one-line docstring has two sentences, do we split them up with a newline?

Reply:
> Given that this one-line docstring has two sentences, do we split them up with a newline?

Now that I re-read after my knee-jerk reaction, I'm unsure. I've only assumed one-line + capitalized + end w/ a period. @richrines1 wdyt? On the one hand simplicity is nice, but on the other it's perhaps too nit-picky.

Reply:
I agree that sticking to one-line should be sufficient, this was just a minor curiosity about what we can get away with in that one-line haha.

Reply:
It's useful to bring up while we're on the topic -- I've thought about it too.